test/test-poll-close-doesnt-corrupt-stack.c

```diff
@@ -69,9 +68,13 @@ static void NO_INLINE close_socket_and_verify_stack() {
   for (i = 0; i < ARRAY_SIZE(data); i++)
     ASSERT(data[i] == MARKER);
 }
+#endif
 
 
 TEST_IMPL(poll_close_doesnt_corrupt_stack) {
+#ifndef _WIN32
```

Comment:
Can you move the TEST_DECLARE and TEST_ENTRY for this test and poll_closesocket out of the `#ifdef _WIN32` guard in test/test-list.h?  Otherwise it doesn't actually run.